app/services/spree/product_feed_service.rb

```diff
@@ -27,54 +27,39 @@ def description
     end
 
     def url
-      "#{Spree::Store.default.url}/#{product.slug}"
```

Comment:
why the hell did they bother creating @store in the init method if they were just going to call this again 🤷‍♂

Reply:
My thoughts exactly.